v2/tools/generator/internal/astmodel/package_import_set.go

```diff
@@ -55,6 +55,15 @@ func (set *PackageImportSet) AddImportsOfReferences(refs ...PackageReference) {
 	}
 }
 
+// AddImportsForPackageReferenceSet ensures this set includes an import of all the references in
+// the specified set.
+// Adding a reference already in the set is fine.
+func (set *PackageImportSet) AddImportsForPackageReferenceSet(refs *PackageReferenceSet) {
```

Comment:
We could make this take `refs iter.Seq[PackageReference]`, which then works with anything that can produce such an iterator, rather than the specific `PackageReferenceSet` type?

Reply:
That would be a good followup. On my list.